release/docker/memgraph_deb.dockerfile

```diff
@@ -1,19 +1,20 @@
 FROM debian:bullseye
 # NOTE: If you change the base distro update release/package as well.
 
-ARG release
+ARG BINARY_NAME
+ARG TARGETARCH
 
 RUN apt-get update && apt-get install -y \
-    openssl libcurl4 libssl1.1 libseccomp2 python3 libpython3.9 python3-pip \
-    --no-install-recommends \
+  openssl libcurl4 libssl1.1 libseccomp2 python3 libpython3.9 python3-pip \
+  --no-install-recommends \
   && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
 RUN pip3 install networkx==2.4 numpy==1.21.4 scipy==1.7.3
 
-COPY ${release} /
+COPY "${BINARY_NAME}${TARGETARCH}.deb" /
 
 # Install memgraph package
-RUN dpkg -i ${release}
+RUN dpkg -i "${BINARY_NAME}${TARGETARCH}.deb"
```

Comment:
First of all, I think `package_docker` and this is not compatible, because `package_docker` passes `${package_name}.${extension}` as binary name, so I think `${BINARY_NAME}${TARGETARCH}.deb` will be expanded to `<memgraph-package-name>.deb<targetarch>.deb`, which is not good.

Is there any reason, why we cannot pass the whole package name as before? I think `TARGET_ARCH` is only used to figure out the installer package.

Reply:
The issue is that `TARGETARCH` is automatically passed by buildx. I've tried running separate build with different arguments but then the images get overriden on dockerhub, and the only way to differentiate architecture is in dockerfile.

Reply:
I've tested the change script and it should work now

Reply:
Ohh, I see. What about unifying all the package names to contain the architecture? So instead of `memgraph.deb` it has to be either `memgraph-amd64.deb` or `memgraph-arm64.deb` (or aarch64, not sure which is the correct).

I know this can be "abused" to work with both `memgraph.deb` and `memgraph-arm64.deb`, but I don't like the concept to use the script in different ways.

Reply:
After the changes I think the current shape is not bad. Changing all the binary names can be a complex task, so let this be like that now.